src/components/Features/style.js

```diff
@@ -0,0 +1,69 @@
+import styled from "styled-components";
+
+export const Container = styled.div`
+    border-radius: 1rem;
+    border: 2px solid ${props => props.theme.primaryLightColor};
+    background: ${props => props.theme.grey191919ToGreyFAFAFA};
+    position: relative;
+    display: flex;
+    align-items: center;
+    overflow: hidden;
+    margin-bottom: 4rem;
+    @media screen and (max-width: 968px) {
+        flex-direction: column-reverse;
+        margin-bottom: 2rem;
+    }
+`;
+
+export const ContentWrapper = styled.div`
+    position: relative;
+    flex: 1;
+    padding: 2rem 1rem 2rem 3rem;
+    .text {
+        color: ${props => props.theme.greyA0AAAAToGrey666666};
+        margin-top: 0.75rem;
+    }
+
+    @media screen and (max-width: 968px) {
+        width: 100%;
+        padding: 0 2rem 2rem;
+    }
+
+    @media screen and (max-width: 500px) {
+        padding: 0 1rem 1rem;
+
+    }
+`;
+export const ImageWrapper = styled.div`
```

Comment:
@ritiksaxena124 it might be nice to offer two slightly different animations to site visitors; to keep each animation fresh.

Reply:
Sounds good to me, I will add the different animation